src/ashley/api/views.py

```diff
@@ -92,8 +92,8 @@ def add_group_moderator(
         # Load current groups
         user_groups = lti_context.get_user_roles(user)
         # Asked to become moderator, check user is not yet moderator and make sure user
-        # belongs to this context by checking he has groups from this context.
-        if _FORUM_ROLE_MODERATOR not in user_groups and len(user_groups) > 0:
+        # belongs to this context by checking he has the group student from this context.
+        if _FORUM_ROLE_MODERATOR not in user_groups and _FORUM_ROLE_STUDENT in user_groups:
```

Comment:
Do you have a test that tries to go around the limitation and proves that it fails?